release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1559,6 +1559,36 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19300
 
 * Previously, an incorrect code was causing `User` and `Group` details to show unrelated subjects. Now, a code has been added to filter by `User` or `Group`, so the `User` and `Group` details show related subjects. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951212[*BZ#1951212*])
 
+* Previously, when you imported a devfile, the Openshift web console ignored the `build guidance` placeholder container because the new deployment had a second container. The web console could not start the container because it could not fetch the placeholder image. As a result, the user container missed the additional configurations. Now, in the web console, the `build guidance` container is dropped from the new deployment, and the container adds the environment variable, ports, and limit configurations. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1952214[BZ#1952214])
```

Comment:
Currently the meaning seems to have changed. 
Suggest: 
Previously, when you imported a devfile, the Openshift web console ignored the `build guidance` placeholder container which provided the configuration for environment variables, ports and limits. The new deployment had a second container which could not start because the placeholder image could not be fetched and it missed the required configurations. Now, the `build guidance` container is dropped from the new deployment, and the container adds the environment variables, ports, and limit configurations.
Please get @sanketpathak to verify this.

Reply:
+1 to what @Preeticp said, that's correct,  the build guidance container is dropped and the console should only keep the environment variables, ports, and limit configurations from this container,

Reply:
+1